ui/packages/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.view.js

```diff
@@ -35,7 +36,94 @@ const createDuplicateQuery = attributes => {
 }
 
 module.exports = Marionette.ItemView.extend({
-  template: template,
+  template() {
+    const disabledExport = store.getSelectedResults().length < 1
+    const exportClass = disabledExport ? 'composed-menu' : ''
+    return (
+      <React.Fragment>
+        <div
+          className="query-interaction interaction-run"
+          title="Run"
+          data-help="Executes the search."
+        >
+          <div className="interaction-text">Run</div>
+        </div>
+        <div
+          className="query-interaction interaction-stop"
+          title="Stop"
+          data-help="Stops a search in the progress of executing.  If results have already returned from some sources, you won't lose them."
+        >
+          <div className="interaction-text">Stop</div>
+        </div>
+        <div
+          className="query-interaction interaction-delete"
+          title="Delete"
+          data-help="Deletes the search."
+        >
+          <div className="interaction-text">Delete</div>
+        </div>
+        <div
+          className="query-interaction interaction-duplicate"
+          title="Duplicate"
+          data-help="Create a new search based off this one."
```

Comment:
✏️ This one has a different verb tense as the others. Create vs creates

Reply:
This isn'y my wording so I'm going to leave it as is.